applications/nrf5340_audio/src/bluetooth/le_audio_cis.h

```diff
@@ -0,0 +1,11 @@
+/*
```

Comment:
I think we can put this in le_audio.h

Reply:
Yea, it is probably better to stick to one .h file.
Can maybe make a comment saying that these defines are only relevant for CIS